tools/scripts/stm32.mk

```diff
@@ -30,6 +43,16 @@ COMPILER_INTELLISENSE_PATH = $(COMPILER_BIN)/arm-none-eabi-gcc
 OPENOCD_SCRIPTS = $(realpath $(addsuffix ..,$(dir $(call rwildcard, $(STM32CUBEIDE)/plugins, *st_scripts/interface/stlink-dap.cfg))))
 OPENOCD_BIN = $(realpath $(dir $(call rwildcard, $(STM32CUBEIDE)/plugins, *bin/openocd)))
 OPENOCD_SVD = $(word 1,$(realpath $(dir $(call rwildcard, $(STM32CUBEIDE)/plugins, *.svd))))
+
+else
+
+OPENOCD_SCRIPTS ?= /usr/share/openocd/scripts
```

Comment:
this path does not exist on my machine, it's distro dependant, also most likely wont exist in Git Bash on windows

Reply:
also from what i understand you're trying to use a generic openocd here, ST has a fork of openocd https://github.com/STMicroelectronics/OpenOCD and the one that comes with the IDE is the forked one